library/spdm_responder_conformance_test_lib/spdm_responder_test_2_capabilities.c

```diff
@@ -623,13 +623,14 @@ void spdm_test_case_capabilities_invalid_request (void *test_context)
                  (SPDM_TEST_VERSION_MASK_V11 | SPDM_TEST_VERSION_MASK_V12)) != 0) {
                 common_test_record_test_message ("test v11 flags - 0x%08x\n",
                                                  invalid_flags_v11[index]);
-                if ((test_buffer->support_version_bitmask & SPDM_TEST_VERSION_MASK_V12) != 0) {
-                    version = SPDM_MESSAGE_VERSION_12;
-                    spdm_request_size = sizeof(spdm_request);
-                } else {
+                if ((test_buffer->support_version_bitmask & SPDM_TEST_VERSION_MASK_V11) != 0) {
```

Comment:
why you need reverse the order of V12 and V11?

Reply:
Thanks. I have changed the code to only check the `(mut_auth_cap == 1) && (encap_cap == 0)` for version1.1 by the PR: #53